### PR TITLE
dataset_structure, validator

### PR DESCRIPTION
**Pull request by fgjdgjdg**

### CODE REVIEW

**Comment by fgjdgjdg** [comment]
Holenyshchenko S.D. KM-93

File: holenyshchenkosd/dataset_insert_manual.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def s(dct, kword, value):
```
Comment (GasperPaul) [inline holenyshchenkosd/dataset_insert_manual.py]: Names, please. How I should deduce what this is doing?

File: holenyshchenkosd/recursion.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+def filter(dct):
```
Comment (GasperPaul) [inline holenyshchenkosd/recursion.py]: I bet this is not printing what you need. You should print not only one attribute, but a whole set if a condition is met.

File: holenyshchenkosd/dataset_insert_manual.py (hunk 1 above)
Comment (GasperPaul) [reply holenyshchenkosd/dataset_insert_manual.py]: fixed in 14aa9b5

**Comment by igortereshchenko** [comment]
невірна назва pull request. Виправити

**Pull request merged**